### PR TITLE
LPS-166139 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-166139

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java [hunk 1]
```diff
@@ -105,6 +106,7 @@ public ObjectDefinitionDeployerImpl(
 		ModelSearchRegistrarHelper modelSearchRegistrarHelper,
 		ObjectDefinitionLocalService objectDefinitionLocalService,
 		ObjectEntryLocalService objectEntryLocalService,
+		ObjectEntryService objectEntryService,
 		ObjectEntryManagerTracker objectEntryManagerTracker,
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java:110]: @marcelabc I think the `objectEntryService` should come after the `objectEntryManagerTracker`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java [hunk 2]
```diff
@@ -127,6 +129,7 @@ public ObjectDefinitionDeployerImpl(
 		_modelSearchRegistrarHelper = modelSearchRegistrarHelper;
 		_objectDefinitionLocalService = objectDefinitionLocalService;
 		_objectEntryLocalService = objectEntryLocalService;
+		_objectEntryService = objectEntryService;
 		_objectEntryManagerTracker = objectEntryManagerTracker;
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java:133]: @marcelabc I think the `objectEntryService` should come after the `objectEntryManagerTracker`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java [hunk 3]
```diff
@@ -138,6 +139,34 @@ public ObjectEntry fetchObjectEntry(long objectEntryId)
...
+		for (ObjectEntry objectEntry : objectEntries) {
+			_checkModelResourcePermission(
+				objectEntry.getObjectDefinitionId(),
+				objectEntry.getObjectEntryId(), ActionKeys.VIEW);
+		}
+
+		return objectEntries;
+	}
+
+	public int getManyToManyObjectEntriesCount(
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java:161]: @marcelabc I don't understand why these new methods don't have the `@Override`  😕

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java [hunk 4]
```diff
@@ -163,14 +192,40 @@ public ObjectEntry getObjectEntry(
...
+		for (ObjectEntry objectEntry : objectEntries) {
+			_checkModelResourcePermission(
+				objectEntry.getObjectDefinitionId(),
+				objectEntry.getObjectEntryId(), ActionKeys.VIEW);
+		}
+
+		return objectEntries;
+	}
+
+	public int getOneToManyObjectEntriesCount(
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java:213]: @marcelabc same as above

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java (hunk 3 above)
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java]: Sorry, I forgot :+1:

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:reevaluate:1257401_5225

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
send to brian: https://github.com/brianchandotcom/liferay-portal/pull/125575

**Pull request closed without merging**